src/js/models/imageReferences.js

```diff
@@ -0,0 +1,20 @@
+import MapList from './MapList.js'
+import ImageReference from './ImageReference.js'
+
+export default class Images extends MapList {
+
+  constructor(
+    sessionID,
+    idKey,
+    ajaxPayload = undefined,
+    attributes = {})
+  {
+    idKey = idKey || 'id'
+    ajaxPayload = ajaxPayload ? ajaxPayload : {transaction: 'getImgOfComb'}
```

Comment:
I like the idea of codifying the request transaction in the models that use them.